src/DynamoWPFCLI/Program.cs

```diff
@@ -139,33 +139,6 @@ private static void RunKeepAlive(StartupUtils.CommandLineArguments cmdLineArgs)
             }
         }
 
-        /// <summary>
-        /// Attempts to import an assembly as a node library from a given file path.
-        /// </summary>
-        /// <param name="model"></param>
-        /// <param name="path"></param>
-        private static void ImportAssembly(DynamoModel model, string path)
```

Comment:
Looks like this function is being called [here](https://github.com/DynamoDS/Dynamo/blob/e19296c1a6aad9aec8ce534f95716beff79f06d9/src/DynamoWPFCLI/Program.cs#L115).

Reply:
You may need to replace that as well with your new startuputils one.